core/src/main/java/hudson/cli/CLICommand.java

```diff
@@ -209,7 +209,23 @@ public String getName() {
      * @param stderr
      *      Connected to the stderr of the CLI client.
      * @return
-     *      Exit code from the command.
+     *      1 If unexpected exception is thrown while performing the command
```

Comment:
see [comments in the JIRA ticket](https://issues.jenkins-ci.org/browse/JENKINS-32273?focusedCommentId=246524&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-246524) for a discusion on why we think 1 is not the best choice here.